### DISCUSSION TITLE
How to turn off the visibility of hidden files?

### DISCUSSION THREAD

**Discussion by sajkofff** [rejetto/hfs#502] [General]
Hello. How to turn off the visibility of hidden files?

**Comment by rejetto** [comment]
what do you mean with "hidden files" ? it can have more than one meaning

**Comment by sajkofff** [comment]
![dont-show-hidden-files](https://github.com/rejetto/hfs/assets/48749271/1f8c48b6-e66e-4c01-b27d-01ff47658268)
For me, hidden Windows files have one meaning))
Especially, temporary MS Office files should not be shown.

**Comment by rejetto** [comment]
quite, but you didn't say "hidden Windows files" in your first message :)

anyway, yes, HFS is supposed to not show files with "hidden" attribute on Windows (other operating systems don't have such thing).
So, it's my understanding that you are seeing in HFS files that are supposed to be hidden.
Can you double check? Pick one file that HFS is displaying, and verify its attributes.

**Comment by sajkofff** [comment]
Perhaps this has something to do with the Cyrillic alphabet? Both office documents are open. Total Commander displays the temporary files of both documents. Temporary files are not shown in Windows 11 Explorer. HFS displays a hidden temporary file in Cyrillic.
![2024-02-27_211332](https://github.com/rejetto/hfs/assets/48749271/641f8fc3-6f3b-45e4-95cd-5d35e36d805e)
![2024-02-27_211356](https://github.com/rejetto/hfs/assets/48749271/2126c436-4846-4ff3-be70-4324b6065a49)
![2024-02-27_211418](https://github.com/rejetto/hfs/assets/48749271/9a4dca4e-d01e-4f64-9cfd-9184bc9804f7)

**Comment by rejetto** [comment]
in your screenshots i cannot see file's attributes. Can you show that?
The fact that it is not displayed in explorer may be caused by other reasons.

**Comment by sajkofff** [comment]
![123](https://github.com/rejetto/hfs/assets/48749271/18d0b0bd-ea39-42a0-8bd2-4645e2448e87)

**Comment by rejetto** [comment]
you were talking about the files starting with ~, right?
but your screenshots seems to be about the OTHER files.

**Comment by sajkofff** [comment]
sorry
![222](https://github.com/rejetto/hfs/assets/48749271/1f774c77-9d3e-48dc-8b79-1eed5bc23995)

**Comment by rejetto** [comment]
ok, i assume the checkbox on the right is the "hidden" (can't read russian).
in your hfs page i see only the file with cyrillic.
is the english hidden correctly?

**Comment by sajkofff** [comment]
yes

**Comment by rejetto** [comment]
ok, we are having a problem with that.
This should fix it
https://github.com/rejetto/hfs/files/14424954/hfs-windows.zip
to install this update you can right-click on the "check for updates" button, and paste this link

**Comment by sajkofff** [comment]
Problem solved. Thank you very much Massimo!

**Comment by rejetto** [comment]
thanks for your feedback

**Comment by anton-polo** [comment]
Hello @rejetto 
I see exactly the same problem with hidden files in Windows in versions 0.52.9 and 0.53.0-rc12

**Reply by anton-polo** [reply]
files with the hidden attribute are displayed to users in HFS

**Reply by rejetto** [reply]
i confirm, i'll open an issue and work on it #637
